odex2apk.py

```diff
@@ -38,7 +38,7 @@
 OAT2DEX = os.getenv("OAT2DEX", os.path.join(_dirname, "oat2dex.jar"))
 
 # Supported architectures (first match will be used)
-architectures = ["arm", "arm64", "x86", "x86_64"]
+architectures = ["x86_64", "x86", "arm64", "arm"]
```

Comment:
For all Nexus images this should work correctly. The Nexis image's APKs are all compiled for 1 architecture only. The 32-bit folder presence on 64-bit images is only to enable compatibility for running third party 32-bit apps on the 64-bit device.